docs/tutorials/02-registries-repositories.md

```diff
@@ -0,0 +1,132 @@
+# Buildah Tutorial 2
+## Using Buildah with container registries
+
+The purpose of this tutorial is to demonstrate how Buildah can be used to move OCI compliant images in and out of private or public registries. 
+
+In the [first tutorial](https://github.com/projectatomic/buildah/blob/master/docs/tutorials/01-intro.md) we built an image from scratch that we called `fedora-bashecho` and we pushed it to a local Docker repository using the `docker-daemon` protocol. We are going to use the same image to push to a private Docker registry.
+
+First we must pull down a registry. As a shortcut we will save the container name, that is returned from the `buildah from` command, into a bash variable called `registry`. This is just like we did in Tutorial 1 :
```

Comment:
nit: I don't think you need the first comma after "container name,".  But thanks for adding the variable explanation.